models/nc-news.models.js

```diff
@@ -218,7 +218,7 @@ exports.removeCommentById = (commentId) => {
     return Promise.reject({ status: 400, msg: "bad request" });
   }
   return db
-    .query(`SELECT * FROM comments`)
+    .query(`SELECT * FROM comments WHERE comment_id = $1`, [commentId])
     .then(({ rows }) => {
       const allActiveComments = rows.map((row) => {
```

Comment:
Again, only a small thing, but if its not there, it would just be an empty array you could check. 
But, if it works, cool, but you're post likely mapping through and empty array again